src/RestClient/endpoints/V3/Themes/ThemeActions.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeAction } from './types';
+
+class ThemeActions {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Downloads a stores Theme.
+   *
+   * @param uuid A valid store Theme UUID
+   * @returns Promise resolving to a single theme
+   */
+  get(uuid: string): ThemeAction['GetResponse'] {
+    return this.client.post(`${themesPath}/${uuid}/actions/download`);
+  }
+
+  /**
+   * Activates a Theme
+   *
+   * @param data Data used to activate a Theme
+   * @returns Promise resolving to an activated Theme
```

Comment:
These JS Doc Blocks should help the developer using this client to understand what might be returned from the API in response to them calling this method. In the case for activate theme, it looks like a 204 Response is sent: https://developer.bigcommerce.com/api-reference/74c84a2dc3574-activate-a-theme

It would be better to explain to the developer that the Promise will resolve to a 204 No Content response